### ISSUE TITLE
Get Snappy Ubuntu Core running inside LXD

### ISSUE THREAD

**Issue by stgraber** [lxc/lxd#1952]
This is a tracking bug similar to the one we had for Docker (#608).

We'd like to get to the point where we can run Snappy inside a LXD container. To get there, we need at the very least:
 - Support for stacked apparmor profile (#1942)
 - Support for unprivileged squashfs (kernel work)
 - Support for unprivileged loop devices (kernel work)

Marking this as blocked on the kernel work.

**Comment by stgraber** [comment]
Closing this as everything except apparmor is done and we're tracking apparmor in a separate issue.
